tensorflow/python/platform/sysconfig.py

```diff
@@ -51,5 +53,32 @@ def get_lib():
   import tensorflow as tf
   return _os_path.join(_os_path.dirname(tf.__file__))
 
+
+def get_compile_flags():
+  """Get the compilation flags for custom operators.
+
+  Returns:
+    The compilation flags.
+  """
+  from tensorflow.python.framework.versions import __cxx11_abi_flag__
```

Comment:
@drpngx is this what you had in mind?

I've tried to put import on top of the module, too - but that causes __cxx11_abi_flag__ to pollute the namespace.

Reply:
You can say `import .. CXX_ABI_FLAG as _CXX_ABI_FLAG` on top.

Reply:
If I do that, it's still there:

>>> dir (tf.sysconfig)
['_CXX11_ABI_FLAG', '__builtins__', '__doc__', '__file__', '__name__', '__package__', '_allowed_symbols', '_os_path', 'get_compile_flags', 'get_include', 'get_lib', 'get_link_flags']

Reply:
Yes, that's fine.

Reply:
Ack, updated.